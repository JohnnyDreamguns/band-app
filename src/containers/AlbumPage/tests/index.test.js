import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import { AlbumPage } from '../index';

const renderer = new ShallowRenderer();

const props = {
  albumId: 2,
  loading: false,
  album: {
    name: 'Album Name',
    released: '2009'
  },
  band: {
    id: 1,
    name: 'Band Name'
  },
  getAlbumById: jest.fn(),
  clearAlbumpage: jest.fn()
};

describe('<AlbumPage />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<AlbumPage {...props} />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
