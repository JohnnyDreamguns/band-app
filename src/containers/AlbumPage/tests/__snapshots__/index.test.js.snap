// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AlbumPage /> should render and match the snapshot 1`] = `
<div>
  <div>
    <h2>
      Album Name
    </h2>
    <p>
      Released: 
      2009
    </p>
    <p>
      Back to 
      <ForwardRef
        to="/band/1"
      >
        Band Name
      </ForwardRef>
    </p>
  </div>
</div>
`;
